akita/src/root.ts

```diff
@@ -1,3 +1,17 @@
 export const isBrowser = typeof window !== 'undefined';
 export const isNotBrowser = !isBrowser;
 export const isNativeScript = typeof global !== 'undefined' && (<any>global).__runtimeVersion !== 'undefined';
+export const hasLocalStorage = () => {
+  try {
+    return typeof localStorage !== 'undefined';
+  } catch (error) {
```

Comment:
Remove the error param from both, please.

Reply:
done